index.js

```diff
@@ -36,6 +36,7 @@ searchBtnCmp.addEventListener("click", function() {
             newWindVal.innerHTML = Math.round(data.wind.speed);
             weatherWindCmp.appendChild(newWindVal);
             let openWeatherIcon = data.weather[0].icon;
+            console.log(openWeatherIcon);
```

Comment:
remove this console.log you don't needed and before you create a pr you need to remove all debuggers/console.logs and commented code